vars/asfMavenTlpPlgnBuild.groovy

```diff
@@ -119,18 +123,25 @@ def doCreateTask( os, jdk, maven, tasks, first, plan, taskContext )
     echo "Skipping ${os}-jdk${jdk} as unsupported by Jenkins Environment"
     return;
   }
+  def recordReporting = false	
   def cmd = [
     'mvn', '-V',
     '-P+run-its',
-    '-Dmaven.test.failure.ignore=true',
     '-Dfindbugs.failOnError=false',
     '-e',
+    taskContext.extraCmd  
   ]
   if (!first) {
     cmd += '-Dfindbugs.skip=true'
 //  } else { // Requires authorization on SonarQube first
 //    cmd += 'sonar:sonar'
+  }	  
+  if (Integer.parseInt(jdk) >= 11 && !taskContext['ciReportingRunned']) {
+    cmd += "-Pci-reporting -Perrorprone" 
```

Comment:
`ci-reporting` , `errorprone` are is some new profiles, I didn't see those before

Reply:
test branch `ci-reporting` with maven-compiler-plugin